internal/api/v2/buckets_test.go

```diff
@@ -0,0 +1,158 @@
+package v2
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/formancehq/go-libs/v3/time"
+	"github.com/formancehq/ledger/internal/controller/system"
+	"github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+)
+
+func TestDeleteBucket(t *testing.T) {
```

Comment:
Please use table driven test as other api tests.
Also, follow the same models than handlers files for tests.